### PR TITLE
Update JS with Add & Remove

### PR DESCRIPTION
**Pull request by ShaliniNayan**
- Create a new JavaScript file for the new functionality.
- Implement a function for adding a new task (add a new element to the array).
- Implement a function for deleting a task (remove an element from the array).
- Implement a function for editing task descriptions.
- By default new tasks should have the property completed set to false and the property index set to the value of the new array length
- Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique.
- All changes to the To Do List should be saved in local storage.

### CODE REVIEW

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @ShaliniNayan, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ No linter errors
✔️ Used GitHub Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 1]
```diff
@@ -1,16 +1,116 @@
...
-    <div class="todo-left">
-      <input type="checkbox" class="checkbox" ${todo.isComplete ? 'checked' : ''}>
-      <label class="todo-text">${todo.description}</label>
-    </div>
-    <button class="options-button"><span class="iconify options" data-icon="mi:options-vertical"></span></button>
-  `;
-  todoList.appendChild(content);
-});
\ No newline at end of file
+// Select the Elements
```
[the remark this answers is not in the document]
Comment (ShaliniNayan) [reply src/index.js:1]: Thanks & Noted

File: src/index.js [hunk 2]
```diff
@@ -1,16 +1,116 @@
...
+const data = localStorage.getItem('TODO');
+
+// clear the local storage
+clear.addEventListener('click', () => {
+  localStorage.clear();
+  window.location.reload();
+});
+
+// Function Add To-Do
+function addToDo(toDo, id, done, trash) {
```
[the remark this answers is not in the document]
Comment (ShaliniNayan) [reply src/index.js:25]: Thanks & Noted.

File: src/index.js [hunk 3]
```diff
@@ -1,16 +1,116 @@
...
+                </li>                
+                `;
+
+  const position = 'beforeend';
+
+  list.insertAdjacentHTML(position, item);
+}
+
+// load the items to user's interface
+function loadList(listArray) {
```
[the remark this answers is not in the document]
Comment (ShaliniNayan) [reply src/index.js:46]: Thanks & Noted.

File: src/index.js [hunk 4]
```diff
@@ -1,16 +1,116 @@
...
+function completeToDo(element) {
+  element.classList.toggle(CHECK);
+  element.classList.toggle(UNCHECK);
+  element.parentNode.querySelector('.text').classList.toggle(LINE_THROUGH);
+
+  LIST[element.id].done = !LIST[element.id].done;
+}
+
+// Remove to do function
+function removeToDo(element) {
```
[the remark this answers is not in the document]
Comment (ShaliniNayan) [reply src/index.js]: Sorry to differ but totally working on my computer.

[the remark this answers is not in the document]
Comment (ShaliniNayan) [reply src/index.js]: Thanks & Noted.

**Review by rloterh — COMMENTED** [review]
Hi Shalini, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Your app cannot run correctly as the supposed index.js file is missing from the project directory. Kindly ensure to include this file and that app runs correctly.

**Kindly remove/update all similar to the underlined:**
image

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Ismailco — APPROVED** [review]
### Hi @ShaliniNayan, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️ Nice UI
✔️ No linter errors
✔️ Used GitHub Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**